src/index.html

```diff
@@ -6,9 +6,70 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          width="40"
+          height="40"
+          class="img"
+          >
+      </a>
+      <nav class="navigation">
+        <ul class="list">
+          <li class="list-item">
+            <a href="#" class="link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="list-item">
+            <a href="#" class="link">
+              Samsung
+            </a>
+          </li>
+          <li class="list-item">
+            <a href="#" class="link">
+              Smartphones
+            </a>
+          </li>
+          <li class="list-item">
+            <a
+              href="#"
+              class="link"
+              data-qa="hover">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="list-item">
+            <a href="#" class="link">
+            Gadgets
```

Comment:
dont forget about formatting;
add two 2 spaces before "Gadgets" ....the same situation we have in every link;
please update)